### PR TITLE
rename chart name to concourse-chart

### PR DESCRIPTION
**Pull request by pivotal-bin-ju**
rename it due to the consistency of folder name and chart name.

when we run the command under `concourse-chart` directory:
```
helm lint .
```
it does require the folder name and chart name to be identical. here is the error message when they mismatched.
```
==> Linting .
[ERROR] Chart.yaml: directory name (concourse-chart) and chart name (concourse) must be the same

Error: 1 chart(s) linted, 1 chart(s) failed
```

This PR is going to fix this issue.

Signed-off-by: Bin Ju <bju@pivotal.io>

### CODE REVIEW

**Comment by taylorsilva** [comment]
I got curious about this requirement and found this issue that recently closed: https://github.com/helm/helm/issues/1979

This constraint on the dir-name and the chart name is actually being removed in helm 3.0: https://github.com/helm/helm/pull/4141

Is there currently a requirement around `helm lint` passing for our chart repo to be considered a real chart repository? Long-term it would be nice to leave the name as `concourse` instead of `concourse-chart`.

Would be interested in @cirocosta's thoughts.

**Comment by cirocosta** [comment]
Hey,

> This constraint on the dir-name and the chart name is actually being removed in helm 3.0: helm/helm#4141

I'd go with keeping `concourse` having that in mind, as Helm 3.0 is almost there 🤔  

Also, if you look at a chart like [harbor](https://github.com/goharbor/harbor-helm/blob/master/Chart.yaml), we can see that the names don't match (same for [vault](https://github.com/hashicorp/vault-helm/blob/master/Chart.yaml)).

tl;dr: I'd keep `concourse` 😁

**Comment by pivotal-bin-ju** [comment]
@taylorsilva and @cirocosta ,
thanks for pointing out the change in v3. 
I totally agree. 
I will close this PR.
